joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -221,6 +246,37 @@ JointTrajectoryController::on_configure(const rclcpp_lifecycle::State & previous
   // TODO(karsten1987): no lifecyle for subscriber yet
   // joint_command_subscriber_->on_activate();
 
+  // State publisher
+  {
+    int state_publish_rate = 50;
+    if (lifecycle_node_->has_parameter("state_publish_rate")) {
```

Comment:
you set a default when declaring the param, there is no need to hardcode the default rate here for a second time nor to check if it exists.

Reply:
Ditto, found a better solution.

Reply:
Fixed in [23e7b0d ](https://github.com/ros-controls/ros2_controllers/pull/25/commits/23e7b0de8c293346934a15ab088579e6a28beea1)